addons/account/static/src/components/document_file_uploader/document_file_uploader.xml

```diff
@@ -1,23 +1,25 @@
 <templates>
 
     <t t-name="account.DocumentFileUploader">
-        <FileUploader
-            acceptedFileExtensions="props.acceptedFileExtensions"
-            fileUploadClass="'document_file_uploader'"
-            multiUpload="true"
-            onUploaded.bind="onFileUploaded"
-            onUploadComplete.bind="onUploadComplete">
-            <t t-set-slot="toggler">
-                <t t-slot="toggler"/>
-            </t>
-            <t t-slot="default"/>
-        </FileUploader>
+        <t t-if="!props.hideUploadButton">
```

Comment:
We should keep upload button only need to hide it with invisible or d-none because if we add condition here it will not render that element and file uploading will not process when user drag and drop a file :thinking: